geps/gep-1897/index.md

```diff
@@ -5,61 +5,53 @@
 
```

Comment:
@candita as a primary stakeholder here: is it your interpretation that the semantic changes here are ultimately compatible with the implementations who have already adopted the current API?

Reply:
@shaneutt no that's not my interpretation.  Furthermore, the language changes are not something that can be inserted or reviewed in a hurry.  I recommend changing the GEP as little as possible instead of overhauling it, so that it can be reviewed more easily.